docs/source/redact/redacting_text.rst

```diff
@@ -50,6 +50,8 @@ This produces the following output:
         "new_text": "[ORGANIZATION_P5XLAH]"
     }
 
+`redact` calls can also be recorded so you can view and analyze results in our UI.  To learn more, read :ref:`record-api-call-section`
```

Comment:
```suggestion
You can also record `redact` calls, so that you can view and analyze results in the Textual application. To learn more, read :ref:`record-api-call-section`
```